app/src/main/java/com/schibsted/tapdadev/CharacterFactory.java

```diff
@@ -0,0 +1,12 @@
+package com.schibsted.tapdadev;
+
+public class CharacterFactory {
+
+    public static Character[] withImages(int... resources) {
```

Comment:
Because it was quicker to write the array. But yeah, the idea is to change it to a list and forget about the index.
One of those things TO-DO ;)

In app/src/main/java/com/schibsted/tapdadev/CharacterFactory.java<https://github.com/scm-spain/tap-da-dev/pull/12#discussion_r55641968>:

> @@ -0,0 +1,12 @@
> +package com.schibsted.tapdadev;
> +
> +public class CharacterFactory {
> +
> +    public static Character[] withImages(int... resources) {


Why Character[] instead List?

-
Reply to this email directly or view it on GitHub<https://github.com/scm-spain/tap-da-dev/pull/12/files#r55641968>.